typespec-tests/src/test/java/com/versioning/added/AddedClientTests.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.versioning.added;
+
+import com.azure.core.util.BinaryData;
+import com.versioning.added.models.EnumV1;
+import com.versioning.added.models.EnumV2;
+import com.versioning.added.models.ModelV1;
+import com.versioning.added.models.ModelV2;
+import org.junit.jupiter.api.Test;
+
+public class AddedClientTests {
+    private final AddedClient addedClient = new AddedClientBuilder()
+            .endpoint("http://localhost:3000").buildClient();
```

Comment:
Do we have to set the endpoint?

Reply:
`reactor.core.Exceptions$ReactiveException: java.net.MalformedURLException: no protocol: /versioning/added/api-version:/interface-v2/v2` exception occurs without endpoint.